src/main/java/edu/greenblitz/tobyDetermined/subsystems/Gyros/GyroInputs.java

```diff
@@ -0,0 +1,11 @@
+package edu.greenblitz.tobyDetermined.subsystems.Gyros;
+
+
+import org.littletonrobotics.junction.AutoLog;
```

Comment:
Do we need a file for GyroInputs and IGyro? shouldn't they be on the same file?

Reply:
@tnrz135

Reply:
I don't love it, but I can live with it being in the same file as the interface.

Reply:
i'll keep the inputs at a separate file